tracer/test/test-applications/debugger/dependency-libs/Samples.Probes.TestRuns/SmokeTests/MultiProbeWithSpanTest.cs

```diff
@@ -0,0 +1,58 @@
+using System.Runtime.CompilerServices;
+using Samples.Probes.TestRuns.Shared;
+
+namespace Samples.Probes.TestRuns.SmokeTests
+{
+    [LogLineProbeTestData(lineNumber: 53)]
+    public class MultiProbeWithSpanTest : IRun
+    {
+        private const string MetricJson = @"{""ref"": ""intArg""}";
+
+        private const string LogJson = @"{
+    ""gt"": [
+      {""ref"": ""myAwesomeLocal""},
+      5
+    ]
+}";
+
+        private const string LogJson2 = @"{
+    ""lt"": [
+      {""ref"": ""intArg""},
+      {""ref"": ""myAwesomeLocal""}
+    ]
+}";
+
+        [MethodImpl(MethodImplOptions.NoInlining)]
+        public void Run()
+        {
+            Method(6);
+        }
+
+        [MethodImpl(MethodImplOptions.NoInlining)]
+        [MetricMethodProbeTestData(
```

Comment:
I think I'm missing something - where are we asserting that the metrics were actually received?

Reply:
Since `captureSnapshot` is marked as `false` there is no snapshot but only sanity assertions. I'll add a test that will also capture a snapshot.
https://github.com/DataDog/dd-trace-dotnet/blob/378425b6aa9318cbf08cfda698e02130bfd5347c/tracer/test/Datadog.Trace.Debugger.IntegrationTests/ProbesTests.cs#L416-L428